<launch>
    <arg name="agent_name"  default="agent8"/>
    <arg name="camera_yaml" default="$(find pixy_camera)/parameter/pixy2camera.yaml" /> 
    <arg name="debug"       default="False"/>
    <arg name="mode"        default="color"/>

    <group ns="$(arg agent_name)">    
        <rosparam file="$(arg camera_yaml)"/>
        <node name="pixy_node" pkg="pixy2_ros" type="pixy_node"  output="screen">
            <param name="camera_yaml"   value="$(arg camera_yaml)"/>
            <param name="debug"         value="$(arg debug)"/>
            <param name="mode"          value="$(arg mode)"/>
        </node>
        
        <node name="pixy_landmark_detection" pkg="pixy2_ros" type="pixy_landmark_detection.py"  output="screen">
            <param name="debug"         value="$(arg debug)"/>
        </node>
    </group>

</launch>
